src/client/ContentAccess.js

```diff
@@ -9,6 +9,7 @@ const objectPath = require("object-path");
 
 const HttpClient = require("../HttpClient");
 const ContentObjectAudit = require("../ContentObjectAudit");
+const bs58 = require("bs58");
```

Comment:
Is this used?

Reply:
forgot to remove them, moved the code to utils. Thanks.